### PR TITLE
Add Editorial for Field Day

### PR DESCRIPTION
**Pull request by ryanchou-dev**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Review by envyaims — APPROVED** [review]
great, i still find this funny how this problem is literally bitmask dp but no one is admitting to it

File: solutions/silver/usaco-1327.mdx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+**Time Complexity:** $\mathcal{O}(2^C \cdot C + NC)$
+
+<LanguageSection>
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+const int BIG = 1e9;  // arbitrarily large integer
```
Comment (SansPapyrus683) [inline solutions/silver/usaco-1327.mdx:57]: why not just int32_max & float('inf')

Comment (ryanchou-dev) [reply solutions/silver/usaco-1327.mdx:57]: overflow -- should i mention that

Comment (SansPapyrus683) [reply solutions/silver/usaco-1327.mdx:57]: i'd like it if you just did a check for if a number was the max value, but that's fine as well

**Pull request merged**